src/vm/i386/asmhelpers.S

```diff
@@ -815,6 +815,13 @@ PATCH_LABEL ExternalMethodFixupPatchLabel
 NESTED_END ExternalMethodFixupStub, _TEXT
 
 #ifdef FEATURE_READYTORUN
+NESTED_ENTRY DynamicHelperArgsStub, _TEXT, NoHandler
+    CHECK_STACK_ALIGNMENT
+    call  eax
+    add   esp, 12
+    ret
+NESTED_END DynamicHelperArgsStub, _TEXT
```

Comment:
@parjong ok, I wasn't sure if we already treat them as cdecl. But what I really meant was that the JIT would know that the stub would add one or two arguments and cleanup the stack after the call to the stub with that knowledge in mind. Since the only purpose of the DynamicHelperArgsStub is to cleanup the extra arguments after the call, the stub would not be needed this way.
However, I don't know how complicated change that would be on the JIT side and if the JIT has a knowledge about these arguments at all.
We could create a github issue for future investigation of this possibility, I would not do that now.